editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -43,15 +46,26 @@ function InvalidBlockWarning( { ignoreInvalid, switchToDefaultType } ) {
 				</Button>
 				{ defaultBlockType && (
 					<Button
-						onClick={ switchToDefaultType }
+						onClick={ switchTo( defaultBlockType ) }
 						isLarge
 					>
 						{
-							/* translators: Revert invalid block to default */
+							/* translators: Revert invalid block to another block type */
 							sprintf( __( 'Convert to %s' ), defaultBlockType.title )
 						}
 					</Button>
 				) }
+
+				{ htmlBlockType && (
+					<Button
+						onClick={ switchTo( htmlBlockType ) }
+						isLarge
+					>
+						{
+							sprintf( __( 'Convert to %s' ), htmlBlockType.title )
```

Comment:
We might want to check the output of `npm run gettext-strings` to see if this is treated as a separate string from the one above, or if we need to explicitly copy the `translators:` comment here to ensure that they're merged.

Reply:
The comment was being removed (probably it takes the last translation into account), so I just added the comment on both. I don't know if it's worth an issue or not?

Reply:
Update: I ended up dropping the duplicated translation. I updated the text with Matìas's suggestion.

Reply:
>I don't know if it's worth an issue or not?

I'm not sure what the expected behavior is, honestly. I think it may be that duplicating is expected, though perhaps if duplication doesn't occur at least including both (not just latest). For core documentation of actions, there's "hints" which allow developers to avoid the duplication, but I don't believe this exists for i18n:

```php
/** This action is documented in wp-admin/admin.php */
```